### DISCUSSION TITLE
Windows Terminal Preview v1.13.1073

### DISCUSSION THREAD

**Discussion by DHowett** [microsoft/terminal#12762] [Announcements]
This is a servicing release to fix a number of big issues in 1.13 preview.

## Changes

This version of Windows Terminal is now distributed in two bundles, one of which works on Windows 10-11 and the
other of which only works on Windows 11. The Windows 11 version is much smaller because we no longer need to work around
a platform issue related to our dependencies.

If you intend on using Terminal as an unpackaged application--that is, extracting the `msix` file--we recommend that
you use the `Win10` bundle. You will need the [Visual C++ runtime redistributable](https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist).

In addition, if you install the _packaged_ version on either Windows 10 or Windows 11, it now depends on [the Visual C++ Universal Runtime Package](https://docs.microsoft.com/en-us/troubleshoot/developer/visualstudio/cpp/libraries/c-runtime-packages-desktop-bridge?msclkid=3e1dd82aac8b11ec8f9dbd86d5979ca8#how-to-install-and-update-desktop-framework-packages).

Despite these distributions having different version numbers, they are built from the same code and there is no
functional difference between them.

If you install the Windows 10 verison on Windows 11, it will probably automatically upgrade itself to the Windows 11
version. It turns out that it is impossible to have two bundles with _the same version number_, so it has to be this
way.

### Usability

* Terminal can once again be configured as a startup application, and can be detected by tools like PowerToys (#12491)
* We are once again usable on `N` (non-media) SKUs of Windows (#12463)
* There was a puzzling "Element not found" error during settings loading; there is no longer such an error (#12687)
* Terminal will no longer mix up profiles when it is launched in response to a console application spawning (#12484)
* Formatted copy will now try harder to preserve Unicode charatcers in RTF (#12586) (thanks @ianjoneill!)
* Displaying multiple dialogs will no longer punch a giant hole in the Terminal (???) (#12625) (#12517)
* You spoke up about the scroll bars being WAY TOO THIN, so we chonked them up (#12608)
* We have replaced the word "Summon" with "Show/Hide" in the command palette for improved localization (#12603)
* Our confidence in the settings UI's _Save_ button has led to us no longer backing up the settings JSON file (#12652)
   * We won't be deleting the 61,000 backups we did leave on your hard drive, but what's a couple thousand kilobytes between friends?

### Appearance

* We've improved the contrast of the tab strip (#12635) (#12529)
* Our iconography has been updated to the Windows 11 style (#12469)
* We have given the issue where acrylic in the titlebar looked _weird_ the heave-ho (#12460)
* Good good new UI fonts have been enabled (Segoe UI Variable) (#12462)

### Accessibility

* Terminal now announces newly-printed text to any attached screen reader (#12358)
* When you delete a profile, we will re-focus the delete button automatically (#12558)
* Command palette search now tries to announce the number of results to the screen reader (#12429)

### Reliability

* We won't crash any longer if you give us a command line that is a directory (#12538) (thanks @ianjoneill!)
* Fixed a crash setting the hotkey during teardown (#12580)
* Fixed a different pair of crashes, also likely related to default terminal handoff (#12666)
* `ScrollConsoleScreenBuffer` no longer takes the console upstate (#12669)
* Pressing <kbd>Page Up</kbd> or <kbd>Page Down</kbd> with an empty command palette, which seemed like a reasonable thing to do, was taught to _not_ crash the Terminal (#12528)

### Rendering

* Font axes/features once again work across a DPI change (#12492)
* AtlasEngine: Fix ConstBuffer invalidation for background color changes (#12667)
* AtlasEngine: Fix inverted cursor alpha (#12548)
* On Windows 10, you should see fewer "couldn't find Cascadia Mono, even though it is RIGHT NEXT TO US" dialogs (#12554)

**Comment by 5HT** [comment]
When proper UTF-8 ?

![зображення](https://user-images.githubusercontent.com/144776/160260353-a5699fb1-a25b-4e98-934d-bcaa5d1a703f.png)

It is 2022 already.

**Reply by zadjii-msft** [reply]
As we mentioned in #10613 - follow #6555 and #8000 for updates. We're planning on having someone take a look at doing the buffer rewrite later this year. That's obviously a LOT of work, which has to both enable proper unicode support but also not break anything, so it'll take time to get it just right.
